src/frontend/views/games_view.py

```diff
@@ -45,8 +45,8 @@ def get_context_data(self, **kwargs):
       return context
 
 
-class GamesListView(ListView):
-  template_name = "pages/games_list.html"
+class GamesListByNetworkView(ListView):
+  template_name = "pages/games_list_by_network.html"
   paginate_by = 100
```

Comment:
By the way do you know the diff between offset and cursor pagination ?

https://use-the-index-luke.com/no-offset

I think we should use cursor based pagination everywhere with a reasonable size if we want to improve performance 

https://github.com/photocrowd/django-cursor-pagination

Reply:
I think I will leave this as todo for future work, since the current thing works okay. This is frontend-only, so that makes it easy to test and change in the future.